openhcl/openhcl_boot/src/main.rs

```diff
@@ -508,6 +511,18 @@ const fn zeroed<T: FromZeroes>() -> T {
     unsafe { core::mem::MaybeUninit::<T>::zeroed().assume_init() }
 }
 
+/// Get the tsc ref time for hw isolated vms in 100ns
+#[allow(unused_variables)]
+fn get_hw_tsc_time(isolation: IsolationType) -> u64 {
```

Comment:
This function should return an `Option<u64>` based on how it's used in other places. I think I'd rather we have a single place for this too so that callers can call it in all cases and we know based on isolation type instead of having the branch later for it's usage.